### PR TITLE
Update Preconfigured-setups.md

### PR DESCRIPTION
**Pull request by litszwaiboris**
- Submitted my setup

### CODE REVIEW

File: pages/Getting Started/Preconfigured-setups.md [hunk 1]
```diff
@@ -36,3 +36,12 @@ Prasanthrangan's dotfiles are a very popular choice that will suit your needs.
 [GitHub](https://github.com/prasanthrangan/hyprdots)
 
 ![Image of prasanthrangan's dotfiles](https://i.ibb.co/W3SYJCc/showcase-2-2412602747.png)
+
+## litszwaiboris 
+
+Want a quick start on getting Hyprland working? Do you love to have a simple productive workflow?
+Litszwaiboris' dotfiles will get you up on your track to Hyprland as fast as possible
```
Comment (fufexan) [inline pages/Getting Started/Preconfigured-setups.md:43]:
```suggestion
Want a quick start on getting Hyprland working? Do you love having a simple productive workflow?
Litszwaiboris' dotfiles will get you up on your track to Hyprland as fast as possible.
```

**Comment by vaxerski** [comment]
Sorry, but no. There are only a few setups there for a reason. All are incredibly popular (lots of support) and very featureful (thats the point)

Please do not submit new ones

**Comment by vaxerski** [comment]
Another thing is that I cannot vouch for these setups so I prefer ones that already get tons of eyes on them. I don't want the wiki to recommend random dotfiles used by no one that might just run rm -rf for a year without anyone noticing

**Comment by litszwaiboris** [comment]
ight understandable

**Pull request closed without merging**